Description: Install libraries and pkg-config file into (multiarch on Ubuntu) libdir
Author: Iain Lane <iain.lane@canonical.com>

Index: b/lib/lib.pro
===================================================================
--- a/lib/lib.pro
+++ b/lib/lib.pro
@@ -60,12 +60,13 @@
 SOURCES += ofonoconnman.cpp
 SOURCES += ofonoconnmancontext.cpp
 
-target.path = $$[QT_INSTALL_PREFIX]/lib
+target.path = $$[QT_INSTALL_LIBS]
 headers.files = $$PUBLIC_HEADERS
 headers.path = $$[QT_INSTALL_PREFIX]/include/ofono-qt
 
 CONFIG += create_pc create_prl
 
+QMAKE_PKGCONFIG_PREFIX = $$[QT_INSTALL_LIBS]
 QMAKE_PKGCONFIG_DESTDIR = pkgconfig
 QMAKE_PKGCONFIG_INCDIR = $$headers.path
 
